styleguide.config.js

```diff
@@ -74,6 +74,10 @@ module.exports = {
             name: "Grid",
             content: "packages/wonder-blocks-grid/docs.md",
             components: "packages/wonder-blocks-grid/components/*.js",
+            ignore: [
+                "packages/wonder-blocks-grid/components/fixed-width-cell.js",
+                "packages/wonder-blocks-grid/components/gutter.js",
+            ],
```

Comment:
Put these in a private section for `Internal Components` like in the section above this one.